net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,130 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
```

Comment:
Explanation: As you can see, `commandSuffix` is the same as `secondaryArchSuffix`, but de facto, right now it ends up being always empty, because x86 on x86_gcc2 is the only secondary architecture we have currently. But maybe that changes in the future. (e.g. when adding 32 bit support on 64-bit systems).